nuxeo-core/nuxeo-core-query/src/main/java/org/nuxeo/ecm/platform/query/api/Aggregate.java

```diff
@@ -80,4 +80,14 @@
      * post filtering.
      */
     List<Bucket> getExtendedBuckets();
+
+    /**
+     * Gets the field name as defined in Nuxeo.
+     * <p>
+     * In Nuxeo the separator for a complex type is the `/` character, in a case where our {@link Aggregate} implementation define a field
+     * as `file:content.mime-type`, this method should returns `file:content/mime-type`.
+     *
+     * @since 11.1
+     */
+    String getNuxeoField();
```

Comment:
`getXPathField` maybe? We're always in Nuxeo so it's not really precise.
@nuxeo/platform WDYT?